kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ScanSuite.scala

```diff
@@ -210,7 +207,9 @@ class ScanSuite extends AnyFunSuite with TestUtils with ExpressionTestUtils with
       lessThanOrEqual(ofInt(2), col("a")), // 2 <= a
       greaterThanOrEqual(ofInt(0), col("a")), // 0 >= a
       not(equals(col("a"), ofInt(1))), // NOT a = 1
-      not(equals(ofInt(1), col("a"))) // NOT 1 = a
+      not(equals(ofInt(1), col("a"))), // NOT 1 = a
+      not(nullSafeEquals(col("a"), ofInt(1))), // NOT a <=> 1
```

Comment:
Can we reverse the arguments here as well? so `NOT 1 <=> a` and `2 <=> a`

Reply:
Done, added two reversed test cases